src/frontend/packages/core/src/components/design-system/Formik/Select/index.tsx

```diff
@@ -1,31 +1,27 @@
+import { Select } from '@openfun/cunningham-react';
 import { useField } from 'formik';
-import { Box, Select, SelectProps, Text } from 'grommet';
+import { Box } from 'grommet';
 import * as React from 'react';
 
-export interface FormikSelectProps extends SelectProps {
+export type FormikSelectProps = Parameters<typeof Select>[0] & {
   name: string;
   label: string;
   changeCallback?: (option: any) => void;
-}
+};
 
 export const FormikSelect = (props: FormikSelectProps) => {
-  const [field] = useField(props.name);
+  const [field, meta, helpers] = useField(props.name);
 
   return (
     <Box gap={'5px'}>
-      <label aria-label={props.label} htmlFor={props.name}>
-        <Text size={'xsmall'} weight={'bold'}>
-          {props.label}
-        </Text>
-      </label>
       <Select
-        aria-label={props.name}
-        size={'small'}
         {...props}
         {...field}
+        text={meta.error ?? props.text}
```

Comment:
Not quit sure we should display `props.text` only when there are error. For example, we should be able to display an help message.